src/prototype_room_creepbuilder.js

```diff
@@ -75,15 +74,15 @@ Room.prototype.inRoom = function(creepMemory, amount = 1) {
   var creepsSpawning = _(this.find(FIND_MY_SPAWNS)).map(s => s.spawning && Game.creeps[s.spawning.name]).compact();
   var creeps = this.find(FIND_MY_CREEPS).concat(creepsSpawning);
   var iMax = creeps.length;
-  if (!iMax) {return false;}
+  if (!iMax) { return false; }
   let j = 0;
   this.memory.roles = this.memory.roles || {};
   for (let i = 0; i < iMax; i++) {
     let iMem = creeps[i].memory;
-    if (!iMem) {continue;}
-    if (creepMemory.role === iMem.role && !(iMem.routing ||
-      (creepMemory.routing.targetRoom === iMem.routing.targetRoom &&
-      creepMemory.routing.targetId === iMem.routing.targetId))) {
+    if (!iMem) { continue; }
+    if (creepMemory.role === iMem.role && (!iMem.routing ||
+        (creepMemory.routing.targetRoom === iMem.routing.targetRoom &&
+          creepMemory.routing.targetId === iMem.routing.targetId))) {
```

Comment:
ho yeah didn't see this modif (i have the same on live, check all for see if there was differences and missed this. thnks x/
the why was for indentation ^^'